src/screens/Sidebar/SidebarScreen.js

```diff
@@ -111,90 +116,88 @@ class SidebarScreen extends React.Component {
-                  <Text style={{ fontFamily: 'OpenSans', color: colors.white, fontSize: 10 }}>
-                    view profile
-                  </Text>
-                </View>
-              </TouchableOpacity>
-            </View>
-          </ProgressiveImage>
-          <SectionList
-            contentContainerStyle={{ paddingBottom: 100 }}
-            sections={sectionListData}
-            keyExtractor={(item, index) => index}
-            ListHeaderComponent={() => <View height={10} />}
-            renderItem={() => <SidebarListItem />}
-            renderSectionHeader={({ section }) => (
-              <SidebarTitle title={section.title} style={{ marginTop: 0 }} />
-            )}
-            renderSectionFooter={({ section }) => {
-              // THIS FUNCTION IS NOT BEING INVOKED !?
-            }}
-            removeClippedSubviews={false}
-            SectionSeparatorComponent={() => <View height={20} />}
-            ListFooterComponent={() => (
-              <TouchableOpacity
-                onPress={this.onLogoutButtonPressed}
-                style={{
-                  marginVertical: 40,
-                  padding: 12,
-                  backgroundColor: colors.white,
-                  alignItems: 'center',
-                }}
-              >
+              <Image
+                style={{ width: 40, height: 40, borderRadius: 20 }}
+                source={(avatar && { uri: avatar.tiny }) || defaultAvatar}
+              />
+              <View style={{ marginLeft: 12, backgroundColor: 'transparent' }}>
                 <Text
-                  style={[
-                    commonStyles.text,
-                    {
-                      fontWeight: '500',
-                      color: colors.activeRed,
-                    },
-                  ]}
+                  style={{
+                    fontFamily: 'OpenSans',
+                    color: colors.white,
+                    fontSize: 14,
+                    fontWeight: '600',
+                  }}
                 >
-                  Log Out
+                  {name}
                 </Text>
-              </TouchableOpacity>
-            )}
-          />
-        </View>
+                <Text style={{ fontFamily: 'OpenSans', color: colors.white, fontSize: 10 }}>
+                  view profile
+                </Text>
+              </View>
+            </TouchableOpacity>
+          </View>
+        </ProgressiveImage>
+        <SectionList
+          contentContainerStyle={{ paddingBottom: 100 }}
+          sections={sectionListData}
+          keyExtractor={(item, index) => index}
+          ListHeaderComponent={() => <View height={10} />}
+          renderItem={() => <SidebarListItem />}
+          renderSectionHeader={({ section }) => (
+            <SidebarTitle title={section.title} style={{ marginTop: 0 }} />
+          )}
+          renderSectionFooter={({ section }) => {
+            // THIS FUNCTION IS NOT BEING INVOKED !?
```

Comment:
make sure you're reading the right docs :P we're on `0.44` which doesn't support this yet :(